### ISSUE TITLE
YouCompleteMe configuration

### ISSUE THREAD

**Issue by aprokop** [ORNL-CEES/DataTransferKit#84]
Would be nice to have a proper working configuration to share. Once we figure it out, we can add a .ycm_extra_conf.py to $DTK_ROOT.

@dalg24 could you share your partial?

**Comment by dalg24** [comment]
Does PR #88 works for you?

**Comment by aprokop** [comment]
No, it does not work. But I first need to make sure that YCM actually works for me in the first place and test it on a small project. Only after that can I make some meaningful comments.

**Comment by dalg24** [comment]
Looks like it.

**Comment by aprokop** [comment]
A small update: only ```:YouCompleteMe GoTo``` does not seem to work. The regular completion seems to be fine.

**Comment by dalg24** [comment]
Hi @wafflebot[bot]
